### ISSUE TITLE
No JSON object could be decoded for instalooter cli

### ISSUE THREAD

**Issue by mdmustafizurrahman** [althonos/InstaLooter#169]
## Library version

*What's the installed library version ? Check with `instalooter --version`*:

```
instalooter 2.0.0
```

## Environment

*Describe here your environment, including:*

* *OS*: Linux 4.13.0-38-generic #43~16.04.1-Ubuntu SMP
* *Python version*: Python 2.7.12
* *`setuptools` version if reporting an issue with installation*: pip 8.1.1
* *non-standard Python implementation if any*

## Error description - runtime

*If you have an issue at runtime, include the required information below:*

### Reproducible test case

*Are you using the CLI ? If so, include a command that can be used to re-raise the 
error, with actual arguments anybody can try:*

```
instalooter user instagram

```

### Expected behaviour

Supposed to down all the post made by account named instagram

### Actual behaviour

*What's actually happening ? Leave empty if asking for a new feature*
```
2018-04-20 17:15:11 user instalooter.cli[10495] NOTICE Starting download of `instagram`
2018-04-20 17:15:11 user instalooter.cli[10495] CRITICAL No JSON object could be decoded
```

**Comment by althonos** [comment]
This was patched in `v2.0.1`.

**Comment by mdmustafizurrahman** [comment]
Great thanks! Working now.
